releases/release-1.31/release-notes/maps/pr-126191-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 126191
+releasenote:
+  text: Reverted `Graduates the `WatchList` feature gate to beta for kube-apiserver
+    and enables `WatchListClient` for kube-controller-manager (KCM)`.
```

Comment:
```suggestion
  text: Reverted "Graduates the `WatchList` feature gate to beta for kube-apiserver
    and enabled `WatchListClient` for kube-controller-manager (KCM)".
```

Reply:
included @sftim's change

Reply:
```suggestion
  text: Reverted "Graduates the `WatchList` feature gate to beta for kube-apiserver
    and enables `WatchListClient` for kube-controller-manager (KCM)".
```
@Checksumz keeping 'enables' as is as its from the previous change log that its now being reverted.